Dockerfile

```diff
@@ -15,16 +15,16 @@ FROM php:$PHP_VERSION
 
 # Install required packages and configure plugins + mods for Chevereto
 RUN apt-get update && apt-get install -y \
-        libgd-dev \
```

Comment:
I'm pretty sure you have to include a tab here. See the sample [Dockerfile from Wordpress image](https://github.com/docker-library/wordpress/blob/master/php7.4/apache/Dockerfile)